src/functions.js

```diff
@@ -114,8 +114,27 @@ export function clearCompleted() {
 
     for (let i = 0; i < todoIds.length; i += 1) {
       todos = todos.filter((todo, index) => index !== todoIds[i]);
+      todos = todos.map((todo, index) => ({
+        value: todo.value,
+        completed: todo.false,
+        index: index + 1,
+
+      }));
+
       renderTodos();
       localStorage.setItem('todos', JSON.stringify(todos));
     }
   });
 }
+export function changeCompletedStatus() {
+  const checked = document.querySelectorAll('.checkbox:checked');
+  if (checked) {
+    todos = todos.map((todo, index) => ({
+      value: todo.value,
+      completed: !todo.false,
+      index: index + 1,
+    }));
+  }
```

Comment:
- [ ] Upon clicking the checkbox, the status shuld be updated from false t;o true in localstorage, kindly fix this
![image](https://user-images.githubusercontent.com/81932478/206866853-65344210-69df-4fd6-a31e-9f7e169b5ec8.png)